README.md

```diff
@@ -8,7 +8,7 @@
 * [Available Serializers](#available-serializers)
 * [Usage](#usage)
 
-The purpose of this library is to transform search string to a intermediary tokenized data structure that can be used to perform analysis on the search or middle ground to translate the query into other DSL or query structures.
+The purpose of this library is to transform a search string to a intermediary tokenized data structure that can be used to perform analysis on the search input or as an intermediate structure to translate the query into other DSLs or query structures.
```

Comment:
My previous request was aimed to remove the first intermediary altogether. Because we use intermediate later in the sentence.